web/src/app/lists/PaginatedList.tsx

```diff
@@ -192,6 +192,8 @@ export function PaginatedList(props: PaginatedListProps): JSX.Element {
     // must be explicitly set when using, in accordance with TS definitions
     const urlProps = item.url && {
       component: AppLink,
+      // NOTE button: false? not assignable to true
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
you can change the comment here to say that `button` must be set to true when using `to`. `button={false}` was displaying as invalid code with all the URL props set (but off) IIRC. you may want to verify with your IDE though.